DSA/3.2 Greedy Algorithms/Reverse delete algorithm for MST.md

```diff
@@ -0,0 +1,180 @@
+## REVERSE DELETE ALGORITHM FOR MINIMUM SPANNING TREE
+
+The reverse-delete algorithm is an algorithm in graph theory used to obtain a minimum spanning tree from a given connected, edge-weighted graph.
+
+In this Algorithm we study about the graph in which we can identify how reverse delete algorithm works to obtain a minimum spanning tree from a given connected, edge-weighted graph.Reverse delete algorithm is opposite to kruskal algorithm. In kruskal algorithm we solve the graph in increasing order and in reverse delete algorithm we solve the graph in decreasing order.Kruskal’s algorithm starts with an empty graph and adds edges while the Reverse-Delete algorithm starts with the original graph and deletes edges from it. If the graph is disconnected, this algorithm will find a minimum spanning tree for each disconnected part of the graph.This algorithm is a greedy algorithm, choosing the best choice given any situation.
+
+The algorithm works as follows:<br>
+ ***Start with a full graph, (V,E) and begin deleting edges in order of decreasing cost as long as deleting it does not disconnect the graph.***
+
+### :arrow_right:Algorithm:-
```

Comment:
No need to add multiple images Make some gap in between and reduce size of images